android/src/main/java/com/mattermost/networkclient/APIClient.kt

```diff
@@ -6,7 +6,7 @@ import okhttp3.Request
 import okhttp3.RequestBody.Companion.toRequestBody
 import okio.IOException
 
-class APIClientModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
+class APIClient(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
```

Comment:
why the change in the class name?

Reply:
Oh happy to leave as `APIClientModule`, it just bugged me a little that we call it `APIClient` whereas `NetworkConstants` is name on both the file / class, and returned via `getName()` so just trying to be more consistent. Happy either way, but not both at the same time 😄

Reply:
I get you, but I think we should keep the RN way of doing things here, for consistency. So we have a Package and a Module file and the name exported to JS is without `Module` in the name

Reply:
Yup! In that case I'll call it `NetworkConstantsModule` for the android side for consistency 👍

Reply:
is `NetworkConstants` a module? or is it just constants? I'm struggling to understand why we have a module named `NetworkConstants`

Reply:
Ah so technically it is just another module; it's initialised the same way as `APIClient` or `GenericClient` is - it just holds a `getConstants()` method which returns an object of constants.

With TurboModules coming - it will be exactly like any of the other methods too; https://reactnative.dev/docs/native-modules-android#exporting-constants